Documentation/operations/upgrade.rst

```diff
@@ -310,240 +310,15 @@ Annotations:
-  Generate the installation script using Cilium CLI >=v0.15.8 to automatically
-  include these parameters.
-* ``enable-endpoint-routes`` now automatically sets ``enable-local-node-route``
-  to false, as local node routes are redundant when per-endpoint routes are enabled.
-* L7 visibility using Pod annotations (``policy.cilium.io/proxy-visibility``) is
-  no longer supported.
-  We recommend users to switch to L7 policies instead (see :ref:`proxy_visibility`).
-* If you are using Gateway API, please make sure that new v1 CRDs are installed.
-  The existing Gateway API resources will continue to work as usual, however, it is
-  better to migrate your resources from v1beta1 to v1 for GatewayClass, Gateway and
-  HTTPRoute resources.
-* The tunnel protocol is no longer automatically set to ``geneve`` when Cilium
-  is configured in native routing mode and :ref:`DSR mode with Geneve` is enabled.
-  Explicitly configure ``--tunnel-protocol=geneve`` (or the equivalent
-  ``tunnelProtocol=geneve`` helm value) when DSR with Geneve is enabled.
-* The ``CILIUM_PREPEND_IPTABLES_CHAIN`` environment variable has been renamed
-  to ``CILIUM_PREPEND_IPTABLES_CHAINS`` (note the trailing ``S``) to more accurately
-  match the name of the associated command line flag ``--prepend-iptables-chains``.
-* ``CiliumNetworkPolicy`` changed the semantics of the empty non-nil slice.
-  For an Ingress CNP, an empty slice in one of the fields ``fromEndpoints``, ``fromCIDR``,
-  ``fromCIDRSet`` and ``fromEntities`` will not select any identity, thus falling back to
-  default deny for an allow policy. Similarly, for an Egress CNP, an empty slice in one of
-  the fields ``toEndpoints``, ``toCIDR``, ``toCIDRSet`` and ``toEntities`` will not select
-  any identity either. Additionally, the behaviour of a CNP with ``toCIDRSet`` or
-  ``fromCIDRSet`` selectors using ``cidrGroupRef`` targeting only non-existent CIDR groups
-  was changed from allow-all to deny-all to align with the new semantics.
-* If Cilium is configured with BPF masquerade support but the requirements are not
-  met (e.g: NodePort service implementation in BPF is disabled or socket load-balancing
-  is disabled), it will fail to initialize and will log an error instead of silently
-  fall back to iptables based masquerading.
-* The ICMP ``type`` field in Network Policy now can be either an ICMP message type integer
-  (for example, ``0`` for Echo Reply), or a corresponding CamelCase message type string (``EchoReply``).
-
 .. _upgrade_cilium_cli_helm_mode:
 
 Cilium CLI
 ~~~~~~~~~~
 
 Upgrade Cilium CLI to `v0.15.0 <https://github.com/cilium/cilium-cli/releases/tag/v0.15.0>`_
 or later to switch to `Helm installation mode <https://github.com/cilium/cilium-cli#helm-installation-mode>`_
-to install and manage Cilium v1.14. Classic installation mode is **not**
-supported with Cilium v1.14.
+to install and manage Cilium v1.17. Classic installation mode is **not**
+supported with Cilium v1.17.
```

Comment:
We should probably take a fresh look at this section and rewrite/remove it. Cilium 1.14 will be the last actively maintained version in a few short weeks, which will mean that we expect all users to be running with Helm installation mode.

Reply:
yea, let me remove it. It seems like install/uninstall for classic mode was removed in v0.16.0 cli version already so it's not even really possible to uninstall with a newer cli version.